packager/packager.cc

```diff
@@ -188,7 +208,7 @@ class PackageInfoImpl final : public core::packages::PackageInfo {
 
     // The other packages to which this package is visible. If this vector is empty, then it means
     // the package is fully public and can be imported by anything.
```

Comment:
Can you update this documentation comment to describe the bool? Or make it apparent from variable/struct naming.